gremlin-server/src/main/java/org/apache/tinkerpop/gremlin/server/auth/AllowAllAuthenticator.java

```diff
@@ -41,7 +42,16 @@ public AuthenticatedUser authenticate(final Map<String, String> credentials) thr
         return AuthenticatedUser.ANONYMOUS_USER;
     }
 
+    /**
+     * @deprecated As of release 3.1.1-incubating, replaced by {@link #newSaslNegotiator(InetAddress)}.
+     * @see <a href="https://issues.apache.org/jira/browse/TINKERPOP3-995">TINKERPOP3-995</a>
```

Comment:
i'm not sure that we need to worry about the redirect dropping - i guess we could discuss a mass find/replace of that stuff. i was going to leave the old stuff as-is in my thinking this morning, but maybe we shouldn't. probably something to discuss on the list.